module-cheatsheets/scRNA-seq-cheatsheet.md

```diff
@@ -48,36 +40,52 @@ In addition to the links above, [Amezquita et al.](https://www.biorxiv.org/conte
 
-| Library/Package      | Piece of Code                           | What it's called                                                              | What it does                                                                                                           |
-|----------------------|-----------------------------------------|-------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|
-| [`alevinQC`](http://www.bioconductor.org/packages/devel/bioc//vignettes/alevinQC/inst/doc/alevinqc.html)             | [`alevinQCReport()`](http://www.bioconductor.org/packages/devel/bioc//vignettes/alevinQC/inst/doc/alevinqc.html#generate-qc-report)                      | Alevin QC Report                                                              | Produces a QC (quality check) report from the alevin output                                                           |
-| [`colorblindr`](https://www.rdocumentation.org/packages/colorblindr/versions/0.1.0)  |        [`scale_color_OkabeIto()`](https://www.rdocumentation.org/packages/colorblindr/versions/0.1.0/topics/scale_colour_OkabeIto)                      | OkabeIto Color Scale    | When added as a layer to a plot, makes the plot colorblind friendly                                                    |
-| [`Rtsne`](https://www.rdocumentation.org/packages/Rtsne/versions/0.15)                | [`Rtsne()`](https://www.rdocumentation.org/packages/Rtsne/versions/0.15/topics/Rtsne)                               | T-Distributed Stochastic Neighbor Embedding using a Barnes-Hut Implementation | Reduces the dimensions of the specified matrix or data frame                                                        |
-| [`tibble`](https://www.rdocumentation.org/packages/tibble/versions/1.4.2)               | [`rownames_to_column()`](https://www.rdocumentation.org/packages/tibble/versions/1.4.2/topics/rownames)      | Rownames to Column    | Transforms the rownames of a data frame into a column (which is added to the start of the data frame)                  |
+| Library/Package|Piece of Code| What it's called| What it does  |
+|----------------|-------------|-----------------|---------------|
+| [`alevinQC`](http://www.bioconductor.org/packages/devel/bioc//vignettes/alevinQC/inst/doc/alevinqc.html) | [`alevinQCReport()`](http://www.bioconductor.org/packages/devel/bioc//vignettes/alevinQC/inst/doc/alevinqc.html#generate-qc-report) | Alevin QC Report | Produces a QC (quality check) report from the alevin output |
+| `dat`| [`DataFrame()`](https://www.rdocumentation.org/packages/dat/versions/0.5.0/topics/DataFrame)| Data frame | Not to be confused with [`data.frame()` from Base R. This is a slightly different data frame-like object needed for storing information in `SingleCellExperiment` object's `colData()`.|
```

Comment:
Looks like `dat` is missing the link to the package documentation.

Reply:
This was also in your original comment, however I do think it's a good idea to link some documentation here as the other packages have links and it may leave the user confused as to why this one does not.

Reply:
Added a link to `dat`.